src/main/resources/assets/mythicmetals/lang/de_de.json

```diff
@@ -0,0 +1,1146 @@
+{
+  "block.alloy_forgery.adamantite_block_forge_controller": "Adamantit Block Schmiede Kontrolleinheit",
```

Comment:
For consistency this should be named in a similar manner to the other Alloy Forgery Forge Controllers.
An example: Bricks Forge Controller - Schmelzensteuerung (Ziegelsteine)